src/App.tsx

```diff
@@ -41,12 +41,13 @@ function ProviderWrapper(): React.ReactElement {
     );
   }
   return (
-    <ThemeProvider
-      initialThemeType={ThemeType.LIGHT}
-    ><StateProvider>
-        <App />
-      </StateProvider>
-    </ThemeProvider>
+    <AppearanceProvider>
```

Comment:
@real0131 It'd be better to go inside `ThemeProvider` because it has a similar personality.

Reply:
Or you can just pass `initialThemeType` to `ThemeProvider` I think.